packages/paypal-commerce-utils/src/paypal-commerce-types.ts

```diff
@@ -93,6 +98,185 @@ export interface PayPalMessagesSdk {
     Messages(options: MessagingOptions): MessagingRender;
 }
 
+export interface PayPalApmSdk {
+    Buttons(options: PayPalCommerceButtonsOptions): PayPalCommerceButtons;
+    PaymentFields(options: PayPalCommercePaymentFieldsOptions): PayPalCommercePaymentFields;
+}
+
+/**
+ *
+ * PayPal Commerce Buttons
+ *
+ */
+export interface PayPalCommerceButtons {
+    render(id: string): void;
+    close(): void;
+    isEligible(): boolean;
+}
+
+export interface PayPalCommerceButtonsOptions {
+    style?: PayPalButtonStyleOptions;
+    fundingSource: string;
+    createOrder(): Promise<string>;
+    onApprove(
+        data: ApproveCallbackPayload,
+        actions: ApproveCallbackActions,
+    ): Promise<boolean | void> | void;
+    onInit?(data: InitCallbackPayload, actions: InitCallbackActions): Promise<void>;
+    onClick?(data: ClickCallbackPayload, actions: ClickCallbackActions): Promise<void> | void;
+    onError?(error: Error): void;
+    onCancel?(): void;
+}
```

Comment:
May I ask you to update all names for interfaces to be able to match that interface is related to PayPalButton.

```suggestion
export interface PayPalCommerceButtonsOptions {
    style?: PayPalButtonStyleOptions;
    fundingSource: string;
    createOrder(): Promise<string>;
    onApprove(
        data: PayPalButtonApproveCallbackPayload,
        actions: PayPalButtonApproveCallbackActions,
    ): Promise<boolean | void> | void;
    onInit?(data: PayPalButtonInitCallbackPayload, actions: PayPalButtonInitCallbackActions): Promise<void>;
    onClick?(data: PayPalButtonClickCallbackPayload, actions: PayPalButtonClickCallbackActions): Promise<void> | void;
    onError?(error: Error): void;
    onCancel?(): void;
}
```

Why? Because there might be another types that can have pretty similar naming and it will be hard to understand which interface used for